Formula/berglas.rb

```diff
@@ -0,0 +1,24 @@
+class Berglas < Formula
+  desc "Tool for managing secrets on Google Cloud"
+  homepage "https://github.com/GoogleCloudPlatform/berglas"
+  url "https://github.com/GoogleCloudPlatform/berglas/archive/v0.2.0.tar.gz"
+  sha256 "69d44c2a0ae0a2d6305d142ecfaf2c52f5f1dd25978af2595fdd1182298b9ee6"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/GoogleCloudPlatform/berglas").install buildpath.children
+
+    cd "src/github.com/GoogleCloudPlatform/berglas" do
+      system "go", "build", "-o", bin/"berglas"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/berglas 2>&1")
+    assert_match "berglas is a CLI tool to reading, writing, and deleting secrets", output
+    assert_match '#{version}', shell_output("#{bin}/berglas --version 2>&1")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I added an extra test - hopefully this is sufficient?